Day-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,2 +1,17 @@
 import React from 'react';
 import './Card.css';
+import CardBanner from './CardBanner';
+import CardContent from './CardContent';
+
+function CardContainer() {
+  return (
+    <React.Fragment>
```

Comment:
You don't need a react fragment here as the entire content is wrapped in a `div`. The function can only return one element (it can have any number of elements/components inside), when you absolutely have to return more than one element, that's when you need a react fragment.